components/tdisplays3/display.py

```diff
@@ -28,10 +28,45 @@ def validate_tdisplays3(config):
 )
 
 async def to_code(config):
+    # Add platformio build_flags for the correct TFT_eSPI settings for the T-Display-S3
+    # This allows using current, unpatched versions of TFT_eSPI
+    cg.add_build_flag("-DUSER_SETUP_LOADED=1")
+    cg.add_build_flag("-DST7789_DRIVER=1")
+    cg.add_build_flag("-DINIT_SEQUENCE_3=1")
+    cg.add_build_flag("-DCGRAM_OFFSET")
+    cg.add_build_flag("-DTFT_RGB_ORDER=TFT_RGB")
+    cg.add_build_flag("-DTFT_INVERSION_ON=1")
+    cg.add_build_flag("-DTFT_PARALLEL_8_BIT=1")
```

Comment:
The TFT_eSPI define does not have a value
```suggestion
    cg.add_build_flag("-DTFT_PARALLEL_8_BIT")
```